### ISSUE TITLE
Error getting HostIP when used with proxy options [Windows + VirtualBox]

### ISSUE THREAD

**Issue by praveenkumar** [minishift/minishift#2561]
```

\minishift.exe start --vm-driver virtualbox --http-proxy http://10.76.99.2:3128 --show-libmachine-logs -v5
[...]
&{[-F /dev/null -o PasswordAuthentication=no -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=no -o ControlPath=none docker@127.0.0.1 -o IdentitiesOnly=yes -i C:\Users\lmohanty\.minishift\machines\minishift\id_rsa -p 50045] C:\WINDOWS\System32\OpenSSH\ssh.exe <nil>}
About to run SSH command:
sudo ping -c1 -w1 192.168.99.1
.SSH cmd err, output: exit status 1: PING 192.168.99.1 (192.168.99.1) 56(84) bytes of data.
```

More Info: https://issues.jboss.org/browse/CDK-315

**Comment by LalatenduMohanty** [comment]
Marking it blocker as it is a regression for Virtualbox + Windows setup on proxy.

**Comment by LalatenduMohanty** [comment]
Resolved via #2562
